auto_nag/scripts/bisection_without_regressed_by.py

```diff
@@ -3,7 +3,7 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 import re
-from typing import Dict
+from typing import Dict, List
```

Comment:
We should move to Python 3.10 at some point :)

Reply:
Agreed!